tests/jit.py

```diff
@@ -44,6 +68,18 @@ def test_double_jit(self):
         self.assertAlmostEqual(((y1 - y3) ** 2).sum(), 0)
         self.assertNotEqual(((y1 - y2) ** 2).sum(), 0)
 
+    def test_jit_kwargs(self):
+        x = objax.random.normal((64, 3))
+        kj = objax.Jit(LinearArgs(3, 3))
+        y1 = kj(x, 1)
+        y2 = kj(x, some_args=1)
+        y3 = kj(x, some_args=2)
+        self.assertEqual(y1.tolist(), y2.tolist())
+        self.assertNotEqual(y1.tolist(), y3.tolist())
+        kj = objax.Jit(LinearTrain(3, 3))
+        with self.assertRaises(ConcretizationTypeError):
```

Comment:
This error comes from JAX, but maybe it worth some explanation in Objax docs (for example in in-depth guide about Jit). From user point of view, it might be not intuitive why bool argument does not work, while float works.

Reply:
I think it's already explained in the docs, in the in-depth guide.